weather-component/src/app/weatherDetails/weatherDetails.component.html

```diff
@@ -1,6 +1,6 @@
 <div class="weather-data layout-column align-items-center mt-50">
   <section class="layout-row align-items-center justify-content-center">
-    <label>Enter City: </label>
+    <label>Enter City:{{searchedData.name}} </label>
```

Comment:
@chetancodex  It is not good to put the value here. You can print the value in input text as a placeholder with `*ngIf="searchedData"`.
So if the user entered the city it will show as a placeholder. But the main if it passes the Hackerrank test.